beta/src/content/reference/react/useInsertionEffect.md

```diff
@@ -4,13 +4,13 @@ title: useInsertionEffect
 
 <Pitfall>
 
-`useInsertionEffect` is aimed at CSS-in-JS library authors. Unless you are working on a CSS-in-JS library and need a place to inject the styles, you probably want [`useEffect`](/reference/react/useEffect) or [`useLayoutEffect`](/reference/react/useLayoutEffect) instead.
+`useInsertionEffect` está orientado a autores de librerías CSS-in-JS. A menos que estés trabajando en una librería CSS-in-JS y necesites un lugar donde inyectar los estilos, probablemente busques [`useEffect`](/reference/react/useEffect) o [`useLayoutEffect`](/reference/react/useLayoutEffect) en su lugar.
```

Comment:
- En esta traducción solemos usar «biblioteca» como traducción de *«library»*.
- Creo que no habría daño en traducir CSS-in-JS a CSS-en-JS.
(Esto aplica a otras partes del documento en que se repiten estas situaciones).